src/main/kotlin/org/jetbrains/research/testspark/display/TestSparkDisplayManager.kt

```diff
@@ -41,14 +42,30 @@ class TestSparkDisplayManager {
     /**
      * Fill the panel with the generated test cases.
      */
-    fun display(report: Report, editor: Editor, uiContext: UIContext, language: SupportedLanguage, project: Project, testsExecutionResultManager: TestsExecutionResultManager) {
+    fun display(
+        report: Report,
+        editor: Editor,
+        uiContext: UIContext,
+        language: SupportedLanguage,
+        project: Project,
+        testsExecutionResultManager: TestsExecutionResultManager,
+        generationTool: GenerationTool,
```

Comment:
Yes, we are checking the compilation status only in the feedback cycle since there are no guarantees that LLM will provide compilable code. For EvoSuite, we really on it's own process for generating